android/java/org/chromium/chrome/browser/rewards/tipping/RewardsTippingSuccessContributionFragment.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.rewards.tipping;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.app.Dialog;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.DisplayMetrics;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.FrameLayout;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.google.android.material.bottomsheet.BottomSheetBehavior;
+import com.google.android.material.bottomsheet.BottomSheetDialog;
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+
+import org.chromium.base.Log;
+import org.chromium.chrome.R;
+
+public class RewardsTippingSuccessContributionFragment extends BottomSheetDialogFragment {
+    final public static String TAG_FRAGMENT = "tipping_success_contribution";
+    private static final String AMOUNT_SELECTED = "amount_selected";
+    private static final String TAG = "TippingSuccess";
+
+    public static RewardsTippingSuccessContributionFragment newInstance(double amountSelected) {
+        RewardsTippingSuccessContributionFragment fragment =
+                new RewardsTippingSuccessContributionFragment();
+        Bundle args = new Bundle();
+        args.putDouble(AMOUNT_SELECTED, amountSelected);
+        fragment.setArguments(args);
+        return fragment;
+    }
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setStyle(STYLE_NORMAL, R.style.AppBottomSheetDialogTheme);
+    }
+    @SuppressLint("RestrictedApi")
```

Comment:
I wonder the reason of this suppression, can it be the reason of crashes on older Android versions?

Reply:
`setupDialog` is a restricted android API method, we should use public lifecycle methods like `onCreateView` `onCreate` etc. Been updating wallet code as well in compliance with android API.

Reply:
Resolved this lint error

Reply:
used onCreateDialog